src/attribute-editor/internal.tsx

```diff
@@ -89,7 +106,18 @@ const InternalAttributeEditor = React.forwardRef(
         >
           {addButtonText}
         </InternalButton>
-        {additionalInfo && <AdditionalInfo id={infoAriaDescribedBy}>{additionalInfo}</AdditionalInfo>}
+        {additionalInfo ? (
+          <AdditionalInfo id={infoAriaDescribedBy}>
+            {/* We are hooking on to this LiveRegion to ensure the order of announcement.
```

Comment:
Adding 2 successive `LiveRegion` components causes a race condition and sometimes one announcement cuts off the other (specially on VoiceOver).

This seems to be both a bug in VoiceOver AND a lack of details in the spec (related discussion: https://github.com/w3c/aria/issues/1689).